src/bootstrap/src/lib.rs

```diff
@@ -520,19 +520,20 @@ impl Build {
             return;
         }
 
-        // check_submodule
-        let checked_out_hash =
-            output(helpers::git(Some(&absolute_path)).args(["rev-parse", "HEAD"]));
-        // update_submodules
+        let submodule_git = || helpers::git(Some(&absolute_path));
```

Comment:
I left this closure in since I think it makes the code more readable to abstract this common call away.